### DISCUSSION TITLE
FalconPy does not Show correct data in Discover module

### DISCUSSION THREAD

**Discussion by zjuka** [CrowdStrike/falconpy#972] [Q&A] [answered]
Hello, 
I'm using FalconPy to get data from Discover module. When I filter data by Hostname - I see that some asset is found (total = 1), but 'resources' is empty and does not provide any aid. If I will remove hostname from filter - I will get list of all Windows aids.

Code:
`hostname = 'EEW2'
    response = falcon.query_hosts(offset=100,
                                  filter=f"hostname:'{hostname}'+platform_name:'Windows'"
                                  )['body']`

Result:
`{'meta': {'pagination': {'limit': 100, 'offset': 100, 'total': 1}, 'powered_by': 'discover-api', 'query_time': 0.020667727, 'trace_id': '1c024f6e-17b9-48fa-b32e-89d5f2d6bb94'}, 'resources': []}`

Can you please explain why is happening in FalconPy?

 BR Artur

**Comment by jshcodes** [comment, marked as answer]
Hi @zjuka -

I think the offset of `100` is why you're being returned an empty resources array. (With only one result, this would be 99 records too many.)

**Reply by zjuka** [reply]
> Hi @zjuka -
> 
> I think the offset of `100` is why you're being returned an empty resources array. (With only one result, this would be 99 records too many.)

Thanks a lot, by removing the offset - I got a correct result.

BR
Artur
